pkg/instrumentation/sdk.go

```diff
@@ -132,14 +154,14 @@ func (i *sdkInjector) injectCommonEnvVar(otelinst v1alpha1.Instrumentation, pod
 	return pod
 }
 
-func (i *sdkInjector) injectCommonSDKConfig(ctx context.Context, otelinst v1alpha1.Instrumentation, ns corev1.Namespace, pod corev1.Pod, index int) corev1.Pod {
-	container := &pod.Spec.Containers[index]
-	resourceMap := i.createResourceMap(ctx, otelinst, ns, pod, index)
+func (i *sdkInjector) injectCommonSDKConfig(ctx context.Context, otelinst v1alpha1.Instrumentation, ns corev1.Namespace, pod corev1.Pod, agentIndex int, appIndex int) corev1.Pod {
```

Comment:
Could you add a docstring here explaining the difference between agentIndex and appIndex

Reply:
Added